tutorial/datastructures.po

```diff
@@ -657,9 +658,8 @@ msgid ""
 "To loop over a sequence in sorted order, use the :func:`sorted` function "
 "which returns a new sorted list while leaving the source unaltered. ::"
 msgstr ""
-"Pour faire une boucle selon un certain classement sur une séquence, utilisez "
-"la fonction :func:`sorted`, elle renvoie une nouvelle liste classée sans "
-"altérer la source : ::"
+"Pour faire une boucle selon une séquence triée, utilisez la fonction :func:"
```

Comment:
Je ne suis pas convaincu par "selon une séquence triée", pourquoi pas :
> Pour parcourir les éléments d'une séquence dans un certain ordre, ...

Reply:
Je pense comprendre pourquoi cette lourdeur dans la phrase d'origine, une séquence (itérable) n'est pas un objet pas nature triable. Il faut alors la convertir en liste.
Je peux mettre "Pour faire une boucle selon une séquence ayant un tri, "

Reply:
Pour le moment le "selon" me gène. Pendant que j'y suis "boucle" aussi n'est pas très représentatif de ce qu'on fait ici.

Ton interprétation est juste, une séquence n'est pas forcément triable sur place (un tuple par exemple). Mais pour le coup "séquence ayant un tri" ne convient pas non plus, (par exemple, le tuple n'a "pas de tri").

On parle donc en d'autres mots d'égrainer dans un certain ordre chaque élément d'une collection non-triable.

Je reviens sur ma proposition précédente avec une petite modification :

> Pour parcourir les éléments d'une séquence de manière ordonnée, ...

ou peut être

> Pour parcourir, dans un certain ordre, les éléments d'une séquence, ...